config/utils/utils.ts

```diff
@@ -1,18 +1,20 @@
 import type { Config, SeasonAddresses } from "@bibliothecadao/eternum";
+import localSeasonAddresses from "../../contracts/common/addresses/local.json";
```

Comment:
can you make sure these local files (local.json, manifest_local.json ) are all incorporated into git ? if not it will not build

![image](https://github.com/user-attachments/assets/1320e0bd-89e7-49b4-acbc-4ee23c33e8af)